src/tasks/pollYoutube.js

```diff
@@ -0,0 +1,27 @@
+const { red, green } = require('colors/safe');
+const { schedule } = require('node-cron');
+const { fetchLatestYoutubeVideos } = require('../helpers/fetchData');
+const Video = require('../api/video/video.model');
+
+schedule('59 * * * *', async () => {
+  try {
+    let { date: lastDate } = await Video.findOne({}).sort({ date: -1 });
```

Comment:
let -> const

Reply:
i can´t use const bc i need to change it in the next line

` lastDate = lastDate.toISOString();`

Reply:
This will work: 

```javascript
  const { date: lastDate } = await Video.findOne({}).sort({ date: -1 });
  const fetchedVideos = await fetchLatestYoutubeVideos({ publishedAfter: lastDate.toISOString() });
```

Reply:
I need to use the value again in the if statement below. it's okey to do `lastDate.toISOString()` again? That's the only reason why I didn't write what to propose.

Reply:
You're right. I haven't looked through the code yet. We have a few options:

1. Leave it as it is.
2. Modify the code so we don't need the if statement on [line 16](https://github.com/CodingGardenCommunity/app-backend/blob/f43bb2916a2dcd34c2b4f217a0a0cb55f988cef3/src/tasks/pollYoutube.js#L16).
3. Write it like this: 

```javascript
const { date } = await Video.findOne({}).sort({ date: -1 });
const lastDate = date.toISOString();
```